app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryEffectHandler.kt

```diff
@@ -2,27 +2,30 @@ package org.simple.clinic.bp.entry
 
 import com.spotify.mobius.rx2.RxMobius
 import io.reactivex.ObservableTransformer
+import org.simple.clinic.util.UserClock
 import org.simple.clinic.util.UserInputDatePaddingCharacter
 import org.simple.clinic.util.scheduler.SchedulersProvider
-import org.threeten.bp.LocalDate
+import org.simple.clinic.util.toLocalDateAtZone
 
 object BloodPressureEntryEffectHandler {
   fun create(
       ui: BloodPressureEntryUi,
+      userClock: UserClock,
       inputDatePaddingCharacter: UserInputDatePaddingCharacter,
       schedulersProvider: SchedulersProvider
   ): ObservableTransformer<BloodPressureEntryEffect, BloodPressureEntryEvent> {
     return RxMobius
         .subtypeEffectHandler<BloodPressureEntryEffect, BloodPressureEntryEvent>()
-        .addConsumer(PrefillDate::class.java, { prefillDate(ui, it.date, inputDatePaddingCharacter) }, schedulersProvider.ui())
+        .addAction(PrefillDateForNewEntry::class.java, { prefillDateForNewEntry(ui, userClock, inputDatePaddingCharacter) }, schedulersProvider.ui())
         .build()
   }
 
-  fun prefillDate(
+  private fun prefillDateForNewEntry(
       ui: BloodPressureEntryUi,
-      date: LocalDate,
+      userClock: UserClock,
       inputDatePaddingCharacter: UserInputDatePaddingCharacter
   ) {
+    val date = userClock.instant().toLocalDateAtZone(userClock.zone)
```

Comment:
We generally use `Instant.now(clock)` instead of `clock.instant()`. If we don't have a reason to use the latter, could we stick to the former convention, please?

Reply:
Done.